handler/oauth2/flow_authorize_implicit.go

```diff
@@ -52,7 +52,9 @@ func (c *AuthorizeImplicitGrantTypeHandler) HandleAuthorizeEndpointRequest(ctx c
 	if !ar.GetResponseTypes().ExactOne("token") {
 		return nil
 	}
-
+	if ar.GetResponseMode() == fosite.ResponseModeNone {
```

Comment:
Use `ar.SetDefaultResponseMode`. And also, as per https://github.com/ory/fosite/pull/509/files#r511293832 this handler should not run if `fosite.ResponseModeQuery` and also `FormPost` in my opinion. Please confirm @mitar .

Reply:
I must say I never really understood why we have handler named "implicit". I would have three authorize handlers: one if you asked for response type "token", one if "code", and one if "id_token". And then based on what you asked for, we allow or not allow some response modes.

I think "implicit" is really just a name to easier think about this, but not really something code/logic should know about. No?

Reply:
So this particular implicit handler can only handle `token` - `id_token` is only enabled if we have support for openid enabled (which needs more config). The original idea was that your oauth2 server could simply not allow/handle implicit at all, if you wanted. This might also be interesting for future OAuth 2.1/2.2 specs which might remove implicit flows in favor of PKCE!